app/src/main/java/shoppinglist/com/shoppinglist/ItemComparator.java

```diff
@@ -0,0 +1,22 @@
+package shoppinglist.com.shoppinglist;
+
+import shoppinglist.com.shoppinglist.database.orm.ShoppingItem;
+import shoppinglist.com.shoppinglist.database.orm.ShoppingList;
+
+/**
+ * Created by hsperr on 4/4/15.
+ */
+public class ItemComparator implements java.util.Comparator<ShoppingItem> {
```

Comment:
yeah I think I wanted to use that to sort the shoppinglist if you cross of items but found it annoying, don't quite remember :dancers: